### PR TITLE
Feature - navbar

### PR DESCRIPTION
**Pull request by winqus**
PR for added header navbar components.

Main changes are located in **src/components/**.

Made some configuration file changes.
Also imported css/scss (for navbar), font files, etc. from previous static Renome version.

### CODE REVIEW

**Comment by ilze-dom** [comment]
Fonts are generally kept in /public folder together with images, favicons, etc.

File: src/components/molecules/Cart.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import CartItem from '../atoms/CartItem';
+import NavButton from '../atoms/NavButton';
+
+function Cart({ count, value, icon }) {
```
Comment (Jalal999) [inline src/components/molecules/Cart.jsx:6]: would be better to follow ES6 syntax while creating functional components. for ex. const Cart = (props) => {}

Comment (winqus) [reply src/components/molecules/Cart.jsx:6]: Yes, I would've liked that too, but for some reason ESLINT with Airbnb ruleset doesn't like anything else, except functions. (although it is ok with const = (....) => {....} in pure JS)

Comment (winqus) [reply src/components/molecules/Cart.jsx:6]: https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless

**Review by ilze-dom — APPROVED** [review]

**Pull request merged**